.eslintignore

```diff
@@ -2,4 +2,5 @@
 **/locales/index.js
 cli/config/init.entrypoint.js
 cli/config/init.App.test.js
-
+shell/src/D2App/*
```

Comment:
The files in this directory should be valid, they are "placeholders" not generated by the compiler (they're replaced later)